styles/_jobs.scss

```diff
@@ -16,3 +16,7 @@
 	position: absolute;
 	right: 16px;
 }
+
+.panel-title {
+	text-transform: none;
+}
```

Comment:
Let's opt for the default clay styles for now since we don't have specific designs for now. I will send a new PR and remove this commit